.github/workflows/workflow_test.yaml

```diff
@@ -46,6 +46,14 @@ jobs:
       trivy-image-config: "tests/workflows/integration/test-upload-charm/trivy.yaml"
       self-hosted-runner: true
       self-hosted-runner-label: "edge"
+  integration-with-uv:
```

Comment:
The `python-version` option is not tested. Can  you set it here so that that part of the code is exercised?